expression/function_traits.go

```diff
@@ -40,6 +40,7 @@ var UnCacheableFunctions = map[string]struct{}{
 
 // unFoldableFunctions stores functions which can not be folded duration constant folding stage.
 var unFoldableFunctions = map[string]struct{}{
+	ast.Sysdate:   {},
```

Comment:
Should other time related functions be unfoldable either?

Reply:
https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html Seems that only sysdate returns the time at which the function executes

Reply:
Got it, thanks. LGTM